pom.xml

```diff
@@ -29,12 +29,17 @@
     <module>nexus-repository-apk-it</module>
   </modules>
 
-  <distributionManagement>
-    <snapshotRepository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-  </distributionManagement>
+<scm>
+  <developerConnection>scm:git:git@github.com:bhamail/test-circleci-maven-release.git</developerConnection>
```

Comment:
This doesn't look right to me, it should be for this repo, yeah @bhamail ?

Reply:
Agreed. The "url" is more like this: https://github.com/sonatype-nexus-community/nexus-repository-r/blob/master/pom.xml#L39, but with suffix `nexus-repository-apk.git`.

Reply:
@bhamail thank you! I see my mistake now -- I have made the update in: https://github.com/sonatype-nexus-community/nexus-repository-apk/pull/10/commits/b1e59bcaa2f789602967b0c5c3ebd01dc041f094 , also changed the `snapshotRepository` back to ossrh